File: ProjekatWEB/Server/Models/Igra.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Collections.Generic;

namespace Models
{
    [Table("Igra")]
    public class Igra
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [MaxLength(50)]
        public string Naziv { get; set; }

        [Required]
        [MaxLength(50)]
        public string Zanr { get; set; }
        
        [Required]
        [Range(1980,2022)]
        public int GodinaIzlaska { get; set; }

        [Required]
        [MaxLength(50)]
        public string Developer { get; set; }

        [Required]
        [MaxLength(50)]
        public string Publisher { get; set; }

        public List<Ocena> Ocene { get; set; }

        public List<Prodavnica> Prodavnice { get; set; }

        public List<Nagrada> Nagrade { get; set; } 
    }
}

File: ProjekatWEB/Klijent/Igra.js
export class Igra{

    constructor(id, naziv, zanr, godinaIzlaska, developer, publisher, prosecnaOcena){
        this.id = id;
        this.naziv = naziv;
        this.zanr = zanr;
        this.godinaIzlaska = godinaIzlaska;
        this.developer = developer;
        this.publisher = publisher;
        this.prosecnaOcena = prosecnaOcena;
    }

    crtaj(host){
       
        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML=this.naziv;
        tr.appendChild(el);
        el = document.createElement("td");
        el.innerHTML=this.zanr;
        tr.appendChild(el);
        el = document.createElement("td");
        el.innerHTML=this.godinaIzlaska;
        tr.appendChild(el);
        el = document.createElement("td");
        el.innerHTML=this.developer;
        tr.appendChild(el);
        el = document.createElement("td");
        el.innerHTML=this.publisher;
        tr.appendChild(el);
        el = document.createElement("td");
        el.innerHTML=this.prosecnaOcena;
        tr.appendChild(el);
    }
}